### ISSUE TITLE
How to install / get on calculator

### ISSUE THREAD

**Issue by thomasb20** [MateoConLechuga/Oiram#23]
How do I get oriam on my calculator? I read the readme but the files are missing.

**Comment by mateoconlechuga** [comment]
What? You just transfer them like any other program.

**Comment by CommentedNewline** [comment]
Sigh...

I could give you the link to this article but you would leave another issue "how to click Link"

Download and Install TI Connect CE
In order to send files to your calculator, you are going to need a piece of software called “TI Connect CE”. This software can be downloaded from the following page: [TI Connect CE Software Download](https://education.ti.com/en/products/computer-software/ti-connect-ce-sw)TI-Connect CE Software Screenshot

Download the Calculator Programs
Next up, we need to download the programs you would like to use on your calculator. For the TI-84 Plus CE, you can download [math programs](https://calcplex.com/downloads/ti84plusce/math/), [games](https://calcplex.com/downloads/ti84plusce/games/), and [other hacks](https://www.ti84calcwiz.com/miscellaneous-programs/).CalcPlex TI-84 Plus CE Math Program Library

Ezoic
Extract the Files (if necessary)
Some program downloads come in the form of a .zip file. If this is the case, this means you need to extract the contents of the .zip before we can continue. To do this, open the .zip file and drag all of the files inside onto your Desktop. If the program instead comes in the form of a .8xg or a .8xp file, skip this step!Extracting files from the zip

Send the Program to Your Calculator
Open the TI Connect CE software that you installed earlier. On the sidebar, click on the second option down (the one below the camera), and connect your calculator to your computer using the charging cable. Make sure that your calculator is turned on, and then drag the file(s) into TI Connect CE. Click “SEND”.Send the program using TI Connect CE

Launch the Program
Grab your TI-84 Plus CE calculator and press the “prgm” button. Select the program you would like to open and press enter. Press enter once more to launch it. That’s all there is to it! The program will load up on your calculator and be ready to use.Math program on a TI-84 Plus CE calculator

If everything went as shown, then the program should be installed and ready for use on your TI-84 Plus CE graphing calculator. Don’t forget to share this article with your friends, and have fun with your newfound calculator abilities!

**Comment by CommentedNewline** [comment]
trol warning
